src/components/App.js

```diff
@@ -56,6 +59,18 @@ function App() {
     setTotalPrice(parseInt(total).toFixed(2))
   }
 
+  const emptyShoppingBag = () => {
+    setShoppingBag([])
+  }
+
+  const updateSuccessMessage = (res) => {
+    if (res) {
```

Comment:
Shouldn't the conditional be `res.data.createOrderForm.message === "success"` or something? because we will get a response from server even if there's an error?

Reply:
this makes more sense reading it now, yes you're right